README.adoc

```diff
@@ -8,42 +8,430 @@ The time for the full course is around three to four days.
 
 The material is created with people with zero Rust experience but with a programming background in mind.
 
-## Overview
+Ferrous Systems offers a large Rust curriculum for both beginner and
+advanced Rust developer at all levels. We are the oldest and most often
+chosen Rust training provider.
+
+Ferrous Systems specialises in custom, topic focused workshops for
+enterprises starting moving into Rust. Supplemental to our courses, we
+offer ongoing help and feedback.
+
+Overview
+--------
 
 The materials are presented as a set of small, self-contained lessons on a specific topic.
+Note that lessons might be taken down when deemed outdated and recreated for new audiences.
 Lessons can be arranged into courses exploring various aspects of Rust.
 
-## Building
+Pricing
+~~~~~~~
+
+Ferrous Systems usually charges per trainer day. That allows for easy
+calculation on one-off trainings and a fair and easy discussion about
+the structure of the course. Other agreements are possible. For example,
+that allows clients to quickly change the setup of the group, e.g. by
+adding a person, without changing the contractual setup.
+
+We offer training on any Rust subject and will create material that we
+deem resuable at no cost for the client.
+
+We also offer training directly on clients products, against a fee to
+produce the material.
+
+Group setup
```

Comment:
I'm not sure if the first and last paragraphs of this section are helpful for our customers to understand our offerings. Perhaps remove paragraph one, and replace paragraph three with:

> When selecting groups for training, we recommend that participants have a similar experience level in Rust, and similar goals for using the language when possible. We are experienced with training developers from a wide range of backgrounds and experience levels, and can design a training suitable for your team.

Reply:
Applied manually.